src/components/stepforms/ConditionsEditor/ConditionForm.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div class="condition-form">
+    <input
+      :value="condition.column"
+      class="condition-form__input condition-form__input--column"
+      @input="updateInput('column', $event.target.value)"
+    />
+    <input
+      :value="condition.operator"
+      class="condition-form__input condition-form__input--operator"
+      @input="updateInput('operator', $event.target.value)"
+    />
+    <input
+      :value="condition.value"
+      class="condition-form__input condition-form__input--value"
+      @input="updateInput('value', $event.target.value)"
+    />
+  </div>
+</template>
+
+<script lang="ts">
+import _ from 'lodash';
```

Comment:
```suggestion
import debounce from 'lodash/debounce';
```

and subsequent changes later in the component

Reply:
Actually, reading this code again, I think it's not a good idea to introduce a `debounce` here.
Updating an object is not an expansive operation.
I prefer to let a higher component, which would trigger such operation at each form update, tocarry the debounce logic. (Then, the debounce delay would be chosen regarding the time cost of the operation)   

Thanks for noticing!

Reply:
See d387f33